Client/sdk/core/CCoreInterface.h

```diff
@@ -180,6 +176,11 @@ class CCoreInterface
     virtual void        OnGameTimerUpdate() = 0;
 
     virtual CDiscordManagerInterface* GetDiscordManager() = 0;
+
+    virtual bool SetChatboxCharacterLimit(int charLimit) = 0;
+    virtual void ResetChatboxCharacterLimit() = 0;
+    virtual int  GetChatboxCharacterLimit() = 0;
+    virtual int  GetChatboxMaxCharacterLimit() = 0;
```

Comment:
For further reference: Netcode (presumeably) uses CCoreInterface. If the order is changed it obviously crashes. This shouldn't be an issue in production nightlies though.